src/main/resources/hudson/plugins/jira/JiraSite/config.jelly

```diff
@@ -10,7 +10,7 @@
     <f:checkbox title="${%Use HTTP authentication instead of normal login}" field="useHTTPAuth" />
   </f:invisibleEntry>
   <f:entry>
-     <f:checkbox title="${%Use Bearer authentication instead of Basic authentication}" field="useBearerAuth" />
+     <f:checkbox title="${%Use Bearer authentication instead of Basic authentication (Note: Bearer authentication is only supported in Jira Server, for Jira Cloud leave this unchecked)}" field="useBearerAuth" />
```

Comment:
You can use `description=""` and add relevant translation description into [config.properties](https://github.com/jenkinsci/jira-plugin/blob/master/src/main/resources/hudson/plugins/jira/JiraSite/config.properties), see example in line 6.